docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib
+sphinx_bootstrap_theme
+git
+git+https://github.com/NeuroTechX/moabb
```

Comment:
These dependencies are already in `pyproject.toml`, why keep a separate `requirements.txt`?